scripts/stacklight_monitor_install.sh

```diff
@@ -1,7 +1,19 @@
 #!/bin/bash -x
 
 # StackLight services
-salt -C 'I@elasticsearch:server' state.sls elasticsearch -b 1
+salt -C 'I@elasticsearch:server' state.sls elasticsearch.server -b 1
 salt -C 'I@influxdb:server' state.sls influxdb -b 1
-salt -C 'I@kibana:server' state.sls kibana
-salt -C 'I@grafana:server' state.sls grafana
+salt -C 'I@kibana:server' state.sls kibana.server
+salt -C 'I@grafana:server' state.sls grafana.server -b 1
```

Comment:
Also it probably avoids race conditions to separate the execution of the server and client states.